packages/salesforcedx-vscode-soql/.vscodeignore

```diff
@@ -5,4 +5,16 @@ test/**
 src/**
 **/*.map
 .gitignore
-tsconfig.json
\ No newline at end of file
+tsconfig.json
+.eslint.json
+**/*.ts
+**/tsconfig.json
+*.xml
+**/*.md
+*.vsix
+node_modules/rxjs/src/**
```

Comment:
Why are you explicitly ignoring these modules ?

Reply:
rxjs ships a bunch of pre-baked bundles of it's library that we don't use. Like esm bundles and it's src directories.  I found this article about it and stole that little tidbit.  Looks like it removes 463 files = ~500k

https://tsh.io/blog/reduce-node-modules-for-better-performance/

Reply:
Awesome, we'll use this for the other extensions :)